src/main/java/io/supertokens/storage/mongodb/Queries.java

```diff
@@ -259,22 +253,17 @@ static void deleteAllExpiredSessions(Start start) {
         collection.deleteMany(Filters.lte("expires_at", System.currentTimeMillis()));
     }
 
-    static SessionInfo getSession(Start start, String sessionHandle) {
+    static SessionInfo getSession(Start start, String sessionHandle) throws StorageQueryException {
         MongoDatabase client = ConnectionPool.getClientConnectedToDatabase(start);
         MongoCollection collection = client.getCollection(Config.getConfig(start).getSessionInfoCollection());
 
         Document result = (Document) collection.find(Filters.eq("_id", sessionHandle)).first();
         if (result == null) {
             return null;
         }
-
-        return new SessionInfo(sessionHandle, result.getString("user_id"),
-                result.getString("refresh_token_hash_2"),
-                new JsonParser().parse(result.getString("session_data")).getAsJsonObject(),
-                result.getLong("expires_at"),
-                new JsonParser().parse(result.getString("jwt_user_payload")).getAsJsonObject(),
-                result.getLong("created_at_time"));
-    }
+        SessionInfoLastUpdatedRowMapper.INSTANCE.mapOrThrow(null);
```

Comment:
Not sure about the purpose of `SessionInfoLastUpdatedRowMapper.INSTANCE.mapOrThrow(null);`.

Reply:
It was a testing code for accessing `INSTANCE`. I have removed it now.